.gitignore

```diff
@@ -1,4 +1,5 @@
 *.py[co]
 build/
 dist/
+.eggs
```

Comment:
Why?

Reply:
I think I did something that created it.  I can reproduce it anymore with neither python-build nor invoking setup.py directly so I'll revert this.  Thanks for the catch :smile: